mail_tracking/README.rst

```diff
@@ -0,0 +1,112 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=============
+Mail tracking
+=============
+
+This module shows email notification tracking status for any messages in
+mail thread (chatter). Each notified partner will have an intuitive icon just
+right to his name.
+
+
+Installation
+============
+
+If you're using a multi-database installation (with or without dbfilter option)
+where /web/databse/selector returns a list of more than one database, then
+you need to add ``mail_tracking`` addon to wide load addons list
+(by default, only ``web`` addon), setting ``--load`` option.
+For example, ``--load=web,mail_tracking``
+
+
+Usage
+=====
+
+When user sends a message in mail_thread (chatter), for instance in partner
+form, then an email tracking is created for each email notification. Then a
+status icon will appear just right to name of notified partner.
+
+These are all available status icons:
+
+.. |sent| image:: mail_tracking/static/src/img/sent.png
+   :width: 10px
+
+.. |delivered| image:: mail_tracking/static/src/img/delivered.png
+   :width: 15px
+
+.. |opened| image:: mail_tracking/static/src/img/opened.png
```

Comment:
[I know that feeling.](http://www.sephko.com/2013/07/112.html)

No, [seriously](https://github.com/OCA/social/pull/67/files/129ed1ba89cff54fcee0ab538f2c20093031cea5#diff-9b28e6f1adb58fc99a48502d5946a38dR34).